modules/statistic_module.rb

```diff
@@ -0,0 +1,13 @@
+module Statistics
+  def top_reader(orders, num = 1)
+    orders.group_by(&:reader).sort_by { |_key, value| -value.count }.first(num).to_h.keys
+  end
+
+  def most_popular_books(orders, num = 1)
```

Comment:
`top_reader` and `most_popular_books` has the same code with 1 difference, could we move it to separate private method with param(`:reader` or `:book`)